sources/3.0.0/sections/10-data_product_format.adoc

```diff
@@ -639,10 +639,17 @@ For example, "latitude,longitude". Reverse scan direction along an axis is indic
 
 | 8
 | Interpolation type
-| interpolationType
+| nearestNeighbor
 | 1
 | Enumeration
-| Code value from <<iho-s100,part=10c,table=22>>
+| Value: 1. See code value from <<iho-s100,part=10c,table=22>>
+
+| 9
+| Offset of data point in cell
+| dataOffsetCode
+| 1
+| Enumeration
+| Value: 5 barycenter (centroid) of cell. See code value from <<iho-s100,part=10c,table=10>>
```

Comment:
Why are you bringing this in?
It's an optional attribute that we didn't have before.
What bothers me is the value "Barycenter (centroid)".
From a geometric point of view, the centroid is not identical to the center point.

The center point and the centroid of a plane surface are two different concepts:

- Center point of a plane surface:
        The center point of a plane surface is the point that lies exactly in the middle of the surface. It is the geometric center that divides the surface into two symmetrical halves.

- Centroid of a plane surface:
        The centroid of a plane surface is the point where the entire mass or weight of the surface is concentrated. It is the point where the surface is in balance when it is balanced on a tip.


In our use case, where the cells are squares, the points are identical. 
The question is, what do you want? The center point or the centroid of the surface. If you choose the barycenter (centroid) here (I'm fine with that), it must also match the rest of the text.

In my opinion, the value 6 (center point) is missing in the enumeration.
That's why I said in Singapore: Leave it out, it's optional in the S-100 anyway.

Reply:
We agree that this might be a bit superfluous and with Topi's picture that shows where the grid point is placed, in the center of the cell, this becomes much clearer. We initially added this after dialogue with Raphael in trying to solve the cell area vs node issue. We do not see the need to add this attribute, as long as the cell area definition has been handled in the way we intend.

Reply:
@AnnaWall01 @RohdeBSH 
Does this require a new issue? If so, will @AnnaWall01 create it?

Reply:
I don't need a new issue for it, as long as the text always refers to the same point.

Reply:
I removed the attribute dataoffsetCode as it is optional and not necessary.
Marking this conversation as resolved.